.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,96 @@
+name: Playwright Smoketest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * *' # nightly at 06:00 UTC against main
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+
+jobs:
+  smoketest-nightly:
+    if: github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main')
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      SRP_PHRASE: ${{ secrets.SRP_PHRASE }}
```

Comment:
Nit: Maybe `SMOKE_TEST_SRP` would be a better name? The `P` in `SRP` already means "phrase" so `SRP_PHRASE` is a bit redundant, and doesn't describe its purpose